apps/web/src/styles/7-components/InfoBox/InfoBox.scss

```diff
@@ -0,0 +1,14 @@
+@use "govuk/settings/_colours-applied" as colors;
+
+$pi-govuk-background-colour:  colors.govuk-colour("light-grey");
```

Comment:
You don't need a sass var for this, you can just call the Sass function where it is used.

Reply:
👍